src/main/java/com/zenjava/javafx/maven/plugin/JarMojo.java

```diff
@@ -144,7 +147,31 @@
      *
      * @parameter property="jfx.copyAdditionalAppResourcesToJar" default-value="false"
      */
-    private boolean copyAdditionalAppResourcesToJar = false;
+    protected boolean copyAdditionalAppResourcesToJar = false;
+
+    /**
+     * To skip copying all dependencies, set this to true. Please note that all dependencies will be added to the
+     * manifest-classpath as normal, only the copy-process gets skipped.
+     *
+     * @since 8.8.0
+     *
+     * @parameter property="jfx.skipCopyingDependencies"
+     */
+    protected boolean skipCopyingDependencies = false;
+
+    /**
+     * @since 8.8.0
+     *
+     * @parameter property="jfx.useLibFolderContentForManifestClasspath" default-value="false"
+     */
+    protected boolean useLibFolderContentForManifestClasspath = false;
+
+    /**
+     * @since 8.8.0
+     *
+     * @parameter property="jfx.fixedManifestClasspath" default-value=""
+     */
+    protected String fixedManifestClasspath = null;
```

Comment:
The developers are able set anything as dependency, but might not want all being present on the classpath, especially when having other stuff inside the folder like non-jars.
Normally the dependencies are gathered from the project-property of maven, no dependency-plugin needed here. If users don't know what they are doing here, they shouldn't put their hands on, or live with the broken build.